src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -15,7 +15,7 @@ file that was distributed with this source code.
     {% set results_count = pager ? pager.countResults() : 0 %}
     {% set has_results = results_count > 0 %}
     {% set current_page_results = has_results ? pager.currentPageResults : [] %}
-    {% set visibility_class = 'sonata-search-result-' ~ has_results ? 'show' : show_empty_boxes %}
+    {% set visibility_class = 'sonata-search-result-' ~ (has_results ? 'show' : show_empty_boxes) %}
```

Comment:
We can avoid using string operation and make code more readable.
```suggestion
    {% set visibility_class = has_results ? 
        'sonata-search-result-show' : 
        'sonata-search-result-show_empty_boxes' 
    %}
```

Reply:
I wrote it on phone, too slow ;(

Reply:
show_empty_boxes is a variable so it's still `'sonata-search-result' ~ show_empty_boxes `

Reply:
Oh, anyway better will be use array_merge like in symfony form themes. Mybe in 5.x :)